src/bot/ConsoleBot.js

```diff
@@ -10,8 +10,13 @@ export default class ConsoleBot extends Bot {
   constructor({
     sessionStore,
     fallbackMethods,
-  }: { sessionStore: SessionStore, fallbackMethods?: boolean } = {}) {
-    const connector = new ConsoleConnector({ fallbackMethods });
+    config,
+  }: {
+    sessionStore: SessionStore,
+    fallbackMethods?: boolean,
+    config?: Object,
```

Comment:
I'd prefer not add a `config` object in the config object. Maybe just add a `mockPlatform` key here.